recipes/coin-utils/all/conanfile.py

```diff
@@ -128,7 +128,7 @@ def package(self):
         self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
         with self._build_context():
             autotools = self._configure_autotools()
-            autotools.install()
+            autotools.install(args=["-j1"])
```

Comment:
Is this required? maybe add a comment if it is?

Reply:
it is
https://github.com/conan-io/conan-center-index/issues/5820

Reply:
could be put under the `if macos` condition, but install is usually fast